src/main/java/uk/gov/companieshouse/acsp/models/filing/PersonName.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.companieshouse.acsp.models.filing;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class PersonName {
+
+    @JsonProperty("first_name")
+    private String firstName;
+    @JsonProperty("last_name")
+    private String lastName;
+    @JsonProperty("middle_name")
+    private String middleName;
```

Comment:
These fields need to match with existing model in chips-filing-consumer
https://github.com/companieshouse/chips-filing-consumer/blob/master/src/main/java/uk/gov/ch/services/dataholdingobjects/source/filingtype/PersonName.java